src/components/cluster-common.js

```diff
@@ -77,9 +77,10 @@ export const ClusterStatusMonitor = ({ cluster, onClusterStoppedRunning = _.noop
   const prevStatus = Utils.usePrevious(currentStatus)
 
   useEffect(() => {
-    if (prevStatus === 'Running' && currentStatus !== 'Running') {
+    const runningStatuses = ['Running', 'Updating']
```

Comment:
is there a better place to put this?

Reply:
Since it seems like there are a couple places that share this logic, we could make a little shared function in `cluster-utils.js` that extracts this notion of 'ok to use'